go/src/koding/kites/kloud/provider/marathon/schema.go

```diff
@@ -38,11 +38,11 @@ var (
 // Credential represents credential information
 // that are required to deploy a Marathon app.
 type Credential struct {
-	URL                string `json:"url" bson:"url" hcl:"url"`
-	BasicAuthUser      string `json:"basic_auth_user" bson:"basic_auth_user" hcl:"basic_auth_user"`
-	BasicAuthPassowrd  string `json:"basic_auth_password" bson:"basic_auth_password" hcl:"basic_auth_password"`
-	RequestTimeout     int    `json:"request_timeout" bson:"request_timeout" hcl:"request_timeout"`
-	DeploymentTimepout int    `json:"deployment_timeout" bson:"deployment_timeout" hcl:"deployment_timeout"`
+	URL               string `json:"url" bson:"url" hcl:"url"`
+	BasicAuthUser     string `json:"basic_auth_user" bson:"basic_auth_user" hcl:"basic_auth_user"`
+	BasicAuthPassword string `json:"basic_auth_password" bson:"basic_auth_password" hcl:"basic_auth_password"`
+	RequestTimeout    int    `json:"request_timeout" bson:"request_timeout" hcl:"request_timeout"`
+	DeploymentTimeout int    `json:"deployment_timeout" bson:"deployment_timeout" hcl:"deployment_timeout"`
```

Comment:
It's here just for API completeness. After https://github.com/Banno/terraform-provider-marathon/issues/42 gets solved, I'm going to use it.